### PR TITLE
Feature responsiveness and UX

### PR DESCRIPTION
**Pull request by Musa-Mabasa**
This branch is for app responsiveness and the overall UX completion.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,8 +1,12 @@
 <!DOCTYPE html>
-<html lang="en" class="h-dvh">
+<html lang="en" class="bg-gradient-color">
   <head>
     <meta charset="UTF-8" />
-    <link rel="icon" type="image/svg+xml" href="/vite.svg" />
+    <link
+      rel="icon"
+      type="image/svg+xml"
+      href="https://github.com/DVT-Launchpad-Web-Cadets/Musa-React-Manga-Project/assets/93927619/e43026c6-b599-4c68-be89-bb0b65ee725d"
```
Comment (LesediMoko) [inline index.html:8]: I see that the link to your icon is a link to your github repo of the same project. Maybe consider having a relative link rather

File: src/features/comicReader/components/NextButton.tsx [hunk 2]
```diff
@@ -8,7 +8,11 @@ const NextButton = ({
   nextChapter: NeighborChapter | undefined;
 }) => {
   return (
-    <Link to={`/reader/${nextChapter?.hid}`} replace>
+    <Link
```
Comment (LesediMoko) [inline src/features/comicReader/components/NextButton.tsx:11]: Love the use of aria-labels!

File: src/features/homeFeature/components/BottomContainer.tsx [hunk 3]
```diff
@@ -14,11 +14,11 @@ const BottomContainer = ({
   useMangaListSetters({ topManga, topManhwa, topManhua });
 
   return (
-    <div className="w-full absolute z-40 top-96 rounded-t-2xl bg-gradient-to-tl from-primary-color via-gradient-color to-primary-color flex flex-col grow pb-5">
+    <main className="w-full absolute z-40 top-96 rounded-t-2xl bg-gradient-to-tl from-primary-color via-gradient-color to-primary-color flex flex-col grow pb-5 md:rounded-none md:block md:z-0 md:top-[730px] md:bg-none">
       <TopMangas headerTitle="Top Manga" type="manga" />
       <TopMangas headerTitle="Top Manhwa" type="manhwa" />
       <TopMangas headerTitle="Top Manhua" type="manhua" />
-    </div>
+    </main>
```
Comment (LesediMoko) [inline src/features/homeFeature/components/BottomContainer.tsx:21]: Impressed by the semantic tags 👏

File: types/index.d.ts [hunk 4]
```diff
@@ -0,0 +1 @@
+declare module "*.png";
```
Comment (LesediMoko) [inline types/index.d.ts]: I'm confused as to what this does, a mistake perhaps?

**Review by LesediMoko — APPROVED** [review]
All looks really good, very impressive 👏. Just minor comments concerning the logo and your index.d.ts file. Other than that, great work Musa! 🏄🏽‍♂️

File: types/index.d.ts (hunk 4 above)
Comment (yanevdw) [reply types/index.d.ts]: I think this is to prevent TypeScript errors when importing images, but also not 100% sure.

**Review by yanevdw — APPROVED** [review]
Excellent work! The refactoring, separation of concerns and responsiveness updates look really good.

File: types/index.d.ts (hunk 4 above)
Comment (Musa-Mabasa) [reply types/index.d.ts]: Correct

File: index.html (hunk 1 above)
Comment (Musa-Mabasa) [reply index.html:8]: This is a link to an image on the wiki, not on the source code.

**Pull request merged**